xcube/api/gen/config.py

```diff
@@ -89,7 +88,7 @@ def get_config_dict(config_obj, open_function):
         except ValueError:
             output_region = None
         if output_region is None or len(output_region) != 4:
-            raise ValueError(f'invalid output_region {config_obj.output_region!r}')
+            raise ValueError(f'invalid output_region {config_obj.get("output_region")!r}')
```

Comment:
The value referred to in the error message is a not the object being tested in the `if` condition.